bin/oh-my-install

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# This script is used to allow the user to install plugins
+
+ohmyzsh_dir=$HOME/ohmyzsh
+zshconfig_dir=$HOME/.zsh/configs
+name=$1
+source_file=$ohmyzsh_dir/plugins/$name/$name.plugin.zsh
+
+# Check if oh-my-zsh repository is downloaded in the home directory
+if [ ! -d "$ohmyzsh_dir" ]; then
+    echo "Oh-my-zsh repository is not downloaded in your $HOME directory. Please download it first."
+    exit 1
+fi
+
+# Check if the plugin name is provided
+if [ -n "$1" ]; then
+# Check if the plugin file name exists in the oh-my-zsh plugins directory -> plugins/git/git.plugin.zsh
+  if [ -e "$source_file" ]; then
+      target_file=~/.zshrc
+      # check if the plugin is already installed
+      if ! grep -q "source $source_file" "$target_file"; then
+          ln -s "$source_file" "$zshconfig_dir"
+          echo >&2 Symlink created successfully.
+          echo "source $source_file" >> "$target_file"
```

Comment:
Hello @mike-burns! thank you so much for your comments again! I've been working on this today, but I'm kinda of stuck. I have added what you suggested to match the symlinks and it works 🥳 but for example, when I install the git plugin I'm getting this error` /Users/joseblanco/.zsh/configs/git.plugin.zsh:207: command not found: compdef` 

For what I've seen googling it seems that it is something related to autocompletition. I tried [this solution](https://stackoverflow.com/questions/59058894/why-is-this-compdef-error-message-referring-to) but I'm still getting the error. Have you seen it before? 

Another question is, for example, if I install the plugin for 1password I would get something like `opswd: function definition file not found `. The `opswd` function lives within the 1password plugin folder, but not in the `1password.plugin.zsh`. I tried to symlink directly to the whole folder rather than to just `1password.plugin.zsh` but it's not working 😢 . Any ideas? 

For the `git` plugin there is something similar where the plugin uses a function called `git_current_branch()` that is defined in lib/git.zsh . 

Thank you again 🙏

Reply:
Seems like OMZ has some dependencies that don't match ours. This whole thing might not work...

For example, `compdef` happens after `autoload compinit`, which happens in `‎zsh/configs/post/completion.zsh` -- after any other config is loaded. That might be too late for `git.plugin.zsh`.

I'm surprised you're only seeing this with the symlinks but not with the `source`...

Reply:
@mike-burns thank you again for the above comment.

I think I will give it another thought to find alternatives on how to approach this. For now, I will close this PR. 
If I have a working alternative I will reopen and tag you again 😃 .